src/server/api/controllers/events.js

```diff
@@ -47,7 +47,7 @@ export default class EventsController {
   getEvent = (req, res) => {
```

Comment:
What do you think of instead using the RESTful resource names? i.e. index, show, create, etc?